Lesson9/merve/index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Time-Saving CSS Techniques to Create Responsive Images</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <body>
+      <div class="first-box">
+        <h1>Time-saving CSS techniques to create responsive images</h1>
+        <div class="social-media-profile">
+          <img class="first-img" src="https://miro.medium.com/v2/resize:fill:44:44/1*p3v_yjsMfK5k1oKYDXij7Q.jpeg"
+    alt="Adrien Zaganelli"/>
+          <img class="second-img" src="https://miro.medium.com/v2/resize:fill:24:24/1*MotlWcSa2n6FrOx3ul89kw.png" alt="freecodecamp logo"/>
+        </div>
+        <div class="profile">
+          <a class="link" href="https://medium.com/@adri_zag">Adrien Zaganelli</a>
```

Comment:
```suggestion
          <a class="link" href="https://medium.com/@adri_zag" rel="noopener follow" target="_blank">Adrien Zaganelli</a>
```
Usually, we want links to be open in a new tab so the user does not leave the current page.